integration_tests/integration/personSearch.spec.ts

```diff
@@ -80,7 +36,7 @@ context('Search for a person', () => {
 
   it('can search for a person on probation by CRN with search by name flag', () => {
```

Comment:
Need to drop the "with search by name flag" from the test name

Reply:
resolved